DQMOffline/Trigger/python/BPHMonitor_cff.py

```diff
@@ -3,34 +3,328 @@
-#muoSelection
-Dimuon10_Jpsi_Barrel_BPHMonitoring = hltBPHmonitoring.clone()
-Dimuon10_Jpsi_Barrel_BPHMonitoring.FolderName = cms.string('HLT/BPH/DiMu10_Jpsi_Barrel/')
-Dimuon10_Jpsi_Barrel_BPHMonitoring.numGenericTriggerEventPSet.hltPaths = cms.vstring("HLT_Dimuon10_Jpsi_Barrel")
-Dimuon10_Jpsi_Barrel_BPHMonitoring.muoSelection = cms.string("abs(eta)<1.4 & isPFMuon & isGlobalMuon  & innerTrack.hitPattern.trackerLayersWithMeasurement>5 & innerTrack.hitPattern.numberOfValidPixelHits>0 ")
-
-# HLT_PFMETNoMu120_PFMHTNoMu120_IDTight
-DoubleMu4_3_Jpsi_Displaced_BPHMonitoring = hltBPHmonitoring.clone()
-DoubleMu4_3_Jpsi_Displaced_BPHMonitoring.FolderName = cms.string('HLT/BPH/DoubleMu4_3_Jpsi_Displaced/')
-DoubleMu4_3_Jpsi_Displaced_BPHMonitoring.numGenericTriggerEventPSet.hltPaths = cms.vstring("HLT_DoubleMu4_3_Jpsi_Displaced")
-DoubleMu4_3_Jpsi_Displaced_BPHMonitoring.muoSelection = cms.string("pt>3 & abs(eta)<2.4 & isPFMuon & isGlobalMuon  & innerTrack.hitPattern.trackerLayersWithMeasurement>5 & innerTrack.hitPattern.numberOfValidPixelHits>0 ")
-
-DoubleMu4_JpsiTrk_Displaced_BPHMonitoring = hltBPHmonitoring.clone()
-DoubleMu4_JpsiTrk_Displaced_BPHMonitoring.FolderName = cms.string('HLT/BPH/DoubleMu4_JpsiTrk_Displaced/')
-DoubleMu4_JpsiTrk_Displaced_BPHMonitoring.numGenericTriggerEventPSet.hltPaths = cms.vstring("HLT_DoubleMu4_JpsiTrk_Displaced")
-DoubleMu4_JpsiTrk_Displaced_BPHMonitoring.muoSelection = cms.string("pt>4 & abs(eta)<2.4 & isPFMuon & isGlobalMuon  & innerTrack.hitPattern.trackerLayersWithMeasurement>5 & innerTrack.hitPattern.numberOfValidPixelHits>0 ")
+Dimuon0_Jpsi_tnp = hltBPHmonitoring.clone()
+Dimuon0_Jpsi_tnp.FolderName = cms.string('HLT/BPH/DiMu0_Jpsi_L1_NO_OS_denTrack2/')
+Dimuon0_Jpsi_tnp.tnp = cms.int32(1)
+Dimuon0_Jpsi_tnp.nofset = cms.int32(1)
+Dimuon0_Jpsi_tnp.numGenericTriggerEventPSet.hltPaths = cms.vstring("HLT_Dimuon0_Jpsi_L1_NoOS_v1")
+Dimuon0_Jpsi_tnp.numGenericTriggerEventPSet.l1Algorithms = cms.vstring("DoubleMu0_SQ")
+Dimuon0_Jpsi_tnp.denGenericTriggerEventPSet.hltPaths = cms.vstring("HLT_Mu7p5_Track2_Jpsi_v4")
+Dimuon0_Jpsi_tnp.denGenericTriggerEventPSet.l1Algorithms = cms.vstring("SingleMu5_SQ OR SingleMu7_SQ")
+Dimuon0_Jpsi_tnp.muoSelection_ref = cms.string("pt>7.5 && abs(eta)<2.4 & isPFMuon & isGlobalMuon  & innerTrack.hitPattern.trackerLayersWithMeasurement>5 & innerTrack.hitPattern.numberOfValidPixelHits>0 ")
+Dimuon0_Jpsi_tnp.muoSelection = cms.string("abs(eta)<2.4 & isPFMuon & isGlobalMuon  & innerTrack.hitPattern.trackerLayersWithMeasurement>5 & innerTrack.hitPattern.numberOfValidPixelHits>0 ")
+
+Dimuon0_Upsilon_tnp = hltBPHmonitoring.clone()
+Dimuon0_Upsilon_tnp.FolderName = cms.string('HLT/BPH/DiMu0_Upsilon_L1_NO_OS_denTrack2/')
+Dimuon0_Upsilon_tnp.tnp = cms.int32(1)
+Dimuon0_Upsilon_tnp.nofset = cms.int32(1)
+Dimuon0_Upsilon_tnp.numGenericTriggerEventPSet.hltPaths = cms.vstring("HLT_Dimuon0_Upsilon_L1_NoOS_v1")
```

Comment:
I think there is still a problem in the naming.
It should be HLT_Dimuon0_Upsilon_L1_4p5NoOS_v1